src/styles/_main.scss

```diff
@@ -0,0 +1,13 @@
+html {
+    box-sizing: border-box;
+  }
```

Comment:
I already added this rule to _normalize.scss. Need to delete in one of two places.